### PR TITLE
Adds new grsec install role

### PR DESCRIPTION
**Pull request by conorsch**
The new Ansible role that allows compiling grsec kernel for DigitalOcean hosts. So far, supports Debian 8, but not Ubuntu 14.04.

### CODE REVIEW

**Comment by conorsch** [comment]
@ageis review the changes here, make sure we're in sync? Once you OK it, I'll merge in.

File: ansible/vars.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+package_filename: linux-image-3.14.51-grsec_3.14.51-grsec-10.00.Custom_amd64.deb
+target_os: ubuntu
```
Comment (ageis) [inline ansible/vars.yml]: This `target_os` should default to Debian ideally. Also should add a comment above:
```
# target operating system can be 'debian' (8.x) or 'ubuntu' (14.04.x)
```

Comment (conorsch) [inline ansible/vars.yml]: @ageis resolved in 627e908.

**Comment by ageis** [comment]
Looks great, we just need to resolve the differences between the main playbooks i.e. `build-deb-pkg.yml` and `build-install-grsec.yml`. The former contains the metapackage role. 

I will also include my documentation contributions in a future commit.

**Comment by conorsch** [comment]
@ageis Sounds great, we'll do the docs and the playbook resolution as separate PRs.

**Pull request merged**